.github/workflows/advanced.yml

```diff
@@ -376,26 +376,44 @@ jobs:
       runtime-package-name: ${{ env.PACK_TOOLCHAIN == 'true' && env.RUNTIME_PACKAGE_NAME || '' }}
       runtime-artifact-name: ${{ env.PACK_TOOLCHAIN == 'true' && env.RUNTIME_ARTIFACT_NAME || '' }}
       toolchain-cache-key: ${{ env.PACK_TOOLCHAIN == 'true' && steps.cache-keys.outputs.toolchain-cache-key || '' }}
+      aarch64-w64-mingw32-msvcrt-toolchain-cache-key: ${{ env.aarch64-w64-mingw32-msvcrt-toolchain-cache-key }}
```

Comment:
It should be possible to create matrix job outputs dynamically instead of listing all the variants here explicitly, see e.g. https://github.com/orgs/community/discussions/69904#discussioncomment-7265152 for the reference. I didn't try it myself, though.

Reply:
The first example is not correct. Matrix cannot be used in variable name. The second one is not related to dynamic output. It will stay as it was discussed.

Reply:
Another out-of-the-box solution https://github.com/orgs/community/discussions/26639#discussioncomment-3838014

Reply:
Thanks for sharing. Maybe on the next step we can do some refactoring. For now I prefer to keep original proposal as it was discussed.